plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts

```diff
@@ -231,6 +245,30 @@ export class DocumentLoadInstrumentation extends InstrumentationBase<unknown> {
     }
   }
 
+  private _getConfig(): DocumentLoadInstrumentationConfig {
+    return this._config;
+  }
+  /**
+   * adds custom attributes to root span if configured
+   */
+  private _addCustomAttributes(span: Span) {
+    const applyCustomAttributesOnSpan =
+      this._getConfig().applyCustomAttributesOnSpan;
+    if (applyCustomAttributesOnSpan) {
+      safeExecuteInTheMiddle(
+        () => applyCustomAttributesOnSpan(span),
+        error => {
+          if (!error) {
+            return;
+          }
+
+          this._diag.error('applyCustomAttributesOnSpan', error);
+        },
+        true
+      );
+    }
+  }
+
```

Comment:
Can you also add a section in the README for this package talking about this feature?

Reply:
Readme updated